.evergreen/config.yml

```diff
@@ -77,27 +77,44 @@ functions:
       params:
         working_dir: src
         shell: bash
-        script: |
+        script: >
           ${PREPARE_SHELL}
 
-          # TODO(NODE-5035): Remove when merged - need to replace with branch just for OIDC.
+
+          # TODO(NODE-5035): Remove these 2 lines when PR merged.
+
           rm -rf "${DRIVERS_TOOLS}"
-          git clone --branch DRIVERS-2415 https://github.com/blink1073/drivers-evergreen-tools.git "${DRIVERS_TOOLS}"
+
+          git clone --branch DRIVERS-2415-orch https://github.com/mongodb-labs/drivers-evergreen-tools.git
+          "${DRIVERS_TOOLS}"
+
 
           cd "${DRIVERS_TOOLS}"/.evergreen/auth_oidc
 
+
           # This is a bit confusing but the ec2.assume_role command before
+
           # this task will overwrite these variables to a different value
+
           # than we have set in our evergreen project config. As these are
+
           # now specific to the OIDC ARN, we re-export for the python
+
           # scripts.
+
           export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+
           export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+
```

Comment:
Are the extra line inserts here intentional?

Reply:
No - our script that generates this file is doing this for some reason.